.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
is this root .idea folder necessary with all its files? is the "Smart Village App" folder not enough?

Reply:
Hi @donni106 
You can remove it.
it is present because you created a new work directory so, git will add that